modules/uipath/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "uipath",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Can you give a small description of what the module is used for ? In the near future, i'd like to review all the modules to give them a meaningful description, so we can use that in a future module list